rest-api/flask_api/app.py

```diff
@@ -123,7 +106,8 @@ def fetch_cities():
     cursor = db.cursor(cursor_factory=extras.DictCursor)
     cursor.execute(sql)
     cities = [dict(row) for row in cursor.fetchall()]
-    return jsonify(cities)
+    cleaned_cities = [x for x in cities if not (x.get('city') is None)] #remove null cities
```

Comment:
I think we can actually do this in the same loop. For example,

```
cities = [dict(row) for row in cursor.fetchall() if row.get('city')]
```

Reply:
@StevenZuber ^ (sorry forgot to ping you)

Reply:
Good catch. Much cleaner.